mesonbuild/cmake/interpreter.py

```diff
@@ -558,28 +558,28 @@ def meson_func(self) -> str:
         return target_type_map.get(self.type.upper())
 
     def log(self) -> None:
-        mlog.log('Target', mlog.bold(self.name), f'({self.cmake_name})')
-        mlog.log('  -- artifacts:      ', mlog.bold(str(self.artifacts)))
-        mlog.log('  -- full_name:      ', mlog.bold(self.full_name))
-        mlog.log('  -- type:           ', mlog.bold(self.type))
-        mlog.log('  -- install:        ', mlog.bold('true' if self.install else 'false'))
-        mlog.log('  -- install_dir:    ', mlog.bold(self.install_dir.as_posix() if self.install_dir else ''))
-        mlog.log('  -- link_libraries: ', mlog.bold(str(self.link_libraries)))
-        mlog.log('  -- link_with:      ', mlog.bold(str(self.link_with)))
-        mlog.log('  -- object_libs:    ', mlog.bold(str(self.object_libs)))
-        mlog.log('  -- link_flags:     ', mlog.bold(str(self.link_flags)))
-        mlog.log('  -- languages:      ', mlog.bold(str(self.languages)))
-        mlog.log('  -- includes:       ', mlog.bold(str(self.includes)))
-        mlog.log('  -- sys_includes:   ', mlog.bold(str(self.sys_includes)))
-        mlog.log('  -- sources:        ', mlog.bold(str(self.sources)))
-        mlog.log('  -- generated:      ', mlog.bold(str(self.generated)))
-        mlog.log('  -- generated_ctgt: ', mlog.bold(str(self.generated_ctgt)))
-        mlog.log('  -- pie:            ', mlog.bold('true' if self.pie else 'false'))
-        mlog.log('  -- override_opts:  ', mlog.bold(str(self.override_options)))
-        mlog.log('  -- depends:        ', mlog.bold(str(self.depends)))
-        mlog.log('  -- options:')
+        mlog.debug('Target', mlog.bold(self.name), f'({self.cmake_name})')
+        mlog.debug('  -- artifacts:      ', mlog.bold(str(self.artifacts)))
```

Comment:
I haven't checked what invokes `self.log()` but in general I think these expansive cmake traces are most likely useless other than for debugging, so :+1:.